src/Components/Resource/ResourceBoardView.tsx

```diff
@@ -79,7 +79,7 @@ export default function BoardView() {
       </div>
 
       <BadgesList {...{ appliedFilters, FilterBadges }} />
-      <ScrollingComponent className="mt-4 flex flex-1 items-start overflow-x-scroll px-4 pb-2">
+      <ScrollingComponent className="mt-4 flex flex-1 items-start overflow-x-scroll px-4 pb-2 @container">
```

Comment:
why are we using a container here, we are not using any queries in the child! 🤔

Reply:
this `@container` is for the div inside `src/Components/Resource/ResourceBoard.tsx`
check here: https://github.com/coronasafe/care_fe/pull/6067/files#:~:text=%3Cdiv-,%3Cdiv,className%3D%7BclassNames(,-%22mr%2D2%20h